Firestore/core/src/firebase/firestore/model/field_value.h

```diff
@@ -189,11 +188,16 @@ class FieldValue : public util::Comparable<FieldValue> {
   static FieldValue FromArray(std::vector<FieldValue>&& value);
   static FieldValue FromMap(const Map& value);
   static FieldValue FromMap(Map&& value);
+  static FieldValue FromMap(
+      std::initializer_list<std::pair<std::string, FieldValue>> value);
 
   size_t Hash() const;
 
   util::ComparisonResult CompareTo(const FieldValue& rhs) const;
 
+  std::string ToString() const;
+  friend std::ostream& operator<<(std::ostream& os, const FieldValue& value);
```

Comment:
Same question about `friend`ship here.

Reply:
Same response. In this case moving it out of the class feels messy since there are so many other classes in this file.